src/context/ContextTypes.ts

```diff
@@ -773,7 +783,12 @@ export interface OrganizationObject {
      */
     market?: OrganizationMarket;
     type:    TentacledInteractionType;
-    name?:   string;
+    /**
+     * An optional human-readable name for the organization
+     *
+     * An optional human-readable name for the contact
```

Comment:
For some reason these generated comments are leaking across the types in a couple of places. It's not immediately clear to me why

Reply:
QuickType has a really annoying habit of combining similar entities based on heuristics. You can read more about it here: http://blog.quicktype.io/under-the-hood/  and can see others trying to deal with it here: https://github.com/glideapps/quicktype/issues/1678

Generally, it doesn't cause problems as the types are usually correct. However, it does mess with these docs. I'll take a look as you can sometimes getting it to stop doing it by altering or setting the `title` element...